docs/cli/usage/headless.md

```diff
@@ -26,18 +29,26 @@ The `amplify init` command takes these parameters:
 - `--app`
 
 ### `--amplify`
+
 Contains basic information of the project, it has these keys:
+
 - `projectName`: the name of the project under development
 - `appId`: the Amplify Service project Id (optional, see below)
 - `envName`: the name of your first environment
 - `defaultEditor`: your default code editor
 
 The `appId` parameter is optional and it is used in two use cases.
+<<<<<<< HEAD
```

Comment:
leftover from merge

Reply:
ah weird, just got this updated. We could potentially plug [markdownlint](https://github.com/DavidAnson/markdownlint) into husky. I don't see it in that readme but I know there's a way for us to lock down the supported languages for fenced code blocks to avoid bash/sh clashing in the future.